.circleci/config.yml

```diff
@@ -79,12 +87,16 @@ jobs:
       - run:
           name: Run backend tests
           command: |
-            bash scripts/run_backend_tests.sh --generate_coverage_report --exclude_load_tests
+            bash scripts/run_backend_tests.sh --test_path=core/platform --generate_coverage_report --exclude_load_tests
       - run:
           name: Generate backend coverage report
           command: |
-            sudo pip install codecov
-            codecov
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
```

Comment:
Can we pull out the common lines (here and above, in the frontend tests) in an anchor and then use that anchor?

Reply:
done